.github/workflows/asv-initial.yml

```diff
@@ -0,0 +1,64 @@
+# This is an initial workflow to run airspeed velocity
+name: asv-initial-workflow
+
+on:
+  push:
+    branches: 
+      - asv-ci
+      - master
+jobs:
+  asv-run:
+    runs-on: ubuntu-18.04
```

Comment:
Should we bump this to 20.04?

Reply:
There is a note about this in the Actions doc:
```Note: The Ubuntu 20.04 virtual environment is currently provided as a preview only. The ubuntu-latest YAML workflow label still uses the Ubuntu 18.04 virtual environment.```
I use 18.04 (not `ubuntu-latest`) cause I give this version to ASV in the machine info

Reply:
I would be surprised if it didn't work, don't think there is any drawbacks if it does work.

Reply:
Okay, I'll change version to 20.04